cmd/erasure-utils.go

```diff
@@ -40,11 +41,19 @@ func newHashWriters(diskCount int, algo string) []hash.Hash {
 func newHash(algo string) hash.Hash {
 	switch algo {
 	case "blake2b":
-		return blake2b.New512()
+		h, err := blake2b.New512(nil)
+		if err != nil {
+			panic(fmt.Sprintf("impossible situation: %s", err))
```

Comment:
Then i guess its fine.. we can ignore and add a comment why we are ignoring and why it wouldn't fail ever.